base/main_test_cluster.go

```diff
@@ -271,3 +272,20 @@ func (c *tbpClusterV2) closeCluster(cluster *gocb.Cluster) {
 		c.logger(context.Background(), "Couldn't close cluster connection: %v", err)
 	}
 }
+
+func (c *tbpClusterV2) getMinClusterCompatVersion() int {
+	cluster := getCluster(c.server)
+	defer c.closeCluster(cluster)
+
+	nodesMeta, err := cluster.Internal().GetNodesMetadata(nil)
+	if err != nil {
+		FatalfCtx(context.Background(), "TEST: failed to fetch nodes metadata: %v", err)
+	}
+	minVal := math.MaxInt
+	for _, node := range nodesMeta {
+		if node.ClusterCompatibility < minVal {
+			minVal = node.ClusterCompatibility
```

Comment:
Isn't `ClusterCompatibility` already `min(version)` across all nodes in the cluster? Do we have to check each node?

In other areas we've always just got the first node's cluster compat version for this reason

Reply:
Ah, didn't know that!